actions/src/main/java/com/redhat/service/bridge/actions/ActionTransformer.java

```diff
@@ -0,0 +1,10 @@
+package com.redhat.service.bridge.actions;
+
+import com.redhat.service.bridge.infra.models.actions.BaseAction;
+
+public interface ActionTransformer {
+
+    ActionTransformer IDENTITY = (action, bridgeId, customerId) -> action;
```

Comment:
@kostola Can you add some comment as to the purpose of this `IDENTITY` variable

Reply:
Done :+1: